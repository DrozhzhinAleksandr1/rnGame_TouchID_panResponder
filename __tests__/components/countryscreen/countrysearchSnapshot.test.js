import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { CountrySearch } from '../../../app/components/countryscreen/countrySearch';
import { localization } from '../../../app/common/localization';

it('create snapshot CountrySearch', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<CountrySearch CurrentLanguage={localization.ru}/>);
    expect(toolbarSnapshot).toMatchSnapshot();
});