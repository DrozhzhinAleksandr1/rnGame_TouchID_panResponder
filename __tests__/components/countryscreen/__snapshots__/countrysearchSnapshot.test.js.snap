// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot CountrySearch 1`] = `
<Component
  style={
    Object {
      "borderColor": "#5A6772",
      "borderRadius": 20,
      "borderWidth": 1,
      "flex": 1,
      "flexDirection": "row",
      "height": 40,
      "marginLeft": 15,
      "marginRight": 15,
    }
  }
>
  <TouchableWithoutFeedback
    accessibilityLabel="BtnIconCountryID"
    onPress={[Function]}
    testID="BtnIconCountryID"
  >
    <Component
      style={
        Object {
          "alignItems": "center",
          "height": "100%",
          "justifyContent": "center",
          "width": 56,
        }
      }
    >
      <Icon_search_gray />
    </Component>
  </TouchableWithoutFeedback>
  <TextInput
    accessibilityLabel="SearchInputCountryID"
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Укажите страну"
    placeholderTextColor="#CECECE"
    selectionColor="#5A6772"
    style={
      Object {
        "color": "#FFFFFF",
        "flex": 1,
        "fontFamily": "Roboto-Light",
        "fontSize": 14,
        "marginRight": 10,
        "padding": 0,
      }
    }
    testID="SearchInputCountryID"
    underlineColorAndroid="transparent"
  />
</Component>
`;
